### ISSUE TITLE
Feedback from Peer Review

### ISSUE THREAD

**Issue by YimengXia** [UBC-MDS/wine-origin-prediction#56]
**Data**
- save the raw data explicitly into /data/raw/, after from ucimlrepo import fetch_ucirepo and data = fetch_ucirepo(id=109)

**Report**
- hide the codes at the beginning
- the significance of the observed class imbalance and its potential impact on model performance (create a plot to visualize class imbalance)
-  include captions for each figure, as well as the figure number for reference within the analysis
- comment on what value of hyperparameter C is chosen for the best model
- include more detailed results from the model, such as confusion matrices or ROC curves to better understand its performance
- provide the coefficient values and their corresponding interpretations from the model
- provide an example at the end of the report to illustrate some predictions made by the Logistic Regression
- add some discussions to reinforce how the predictions benefit the industry

**Reproducibility** 
- I wasn’t able to reproduce the report in the container. I successfully opened up the container using docker compose up of setup option 1. However, when I tried to run multiple scripts listed in the data analysis section and tests using pytest, I encountered an error of a module not found. I wonder if the dockerfile copies dependencies properly from the environment.yml file or my user mistakes. I may need some help from the project team to guide me how to run these scripts properly.

**README**
- clearly listed dependencies required for running the project within the local environment using environment.yml. 
- include the dependencies on dockerfile 
- listing out all the required packages and dependencies in the dockerfile

(we received 3 peer reviews so far, will update later)

**Comment by hbandukw** [comment]
I- save the raw data explicitly into /data/raw/, after from ucimlrepo import fetch_ucirepo and data = fetch_ucirepo(id=109) - DONE!  https://github.com/UBC-MDS/wine-origin-prediction/pull/54

**Comment by sean-m-mckay** [comment]
**Report**
    - hide the codes at the beginning | @hbandukw is working on
    - the significance of the observed class imbalance and its potential impact on model performance (create a plot to visualize class imbalance) | low priority
    - include captions for each figure, as well as the figure number for reference within the analysis | high priority @hbandukw 
    - comment on what value of hyperparameter C is chosen for the best model | done already
    - include more detailed results from the model, such as confusion matrices or ROC curves to better understand its performance  | low priority
    - provide the coefficient values and their corresponding interpretations from the model | medium priority @juliaeveritt 
    - provide an example at the end of the report to illustrate some predictions made by the Logistic Regression | low priority
    - add some discussions to reinforce how the predictions benefit the industry | @hbandukw

**Comment by juliaeveritt** [comment]
Other changes:
@sean-m-mckay 
- create a histogram to illustrate class imbalance and comment on how we approached this with our modelling

**Comment by sean-m-mckay** [comment]
Updates
@juliaeveritt - @YimengXia will work on the coefficients interpretation

@hbandukw Is working on updating her tests for accuracy curve function

@sean-m-mckay Upating EDA to show the class imbalance 
@sean-m-mckay Will work on getting Docker working after

Once @hbandukw is done and @YimengXia has updated the report, she will do some final edits on the report to address:
- We need to add F1 or accuracy scores in the text where appropriate.
- Results, discussion and limitations are still incomplete.
